### ISSUE TITLE
Standardize keyboard shortcuts

### ISSUE THREAD

**Issue by Phyks** [ampache/ampache#1225]
Hi,

What about using more standard keyboard shortcuts (or allow the user to use them as an alternative)?

I am thinking in (working in many players, such as `VLC`):

* `Ctrl+→` / `Ctrl+←` for next/previous song
* `Ctrl+↑` / `Ctrl+↓` for volume up/down
* `Space` for play/pause (Or `Ctrl+Space`)

I do not think they would interfer with browsers default keybindings, and such shortcuts are already used by others webapps such as [Jupyter notebook](https://github.com/jupyter/notebook/).

However, according to [this issue](https://github.com/ampache/ampache/issues/636), it seems the player is an external script, and maybe I should report the issue back there?

As a side note, I do not doubt there is a doc for the available shortcuts somewhere, but I could not find it on my instance. Jupyter notebook has an help menu entry for the shortcuts, which display a window with shortcuts over the page: http://www.perfectlyrandom.org/assets/jupyter-keyboard-shortcuts.png. It is really useful and there may be some ideas to take from them :)

**Comment by Phyks** [comment]
Looks like it can be done easily using the `keyBindings` option: http://jplayer.org/latest/developer-guide/#jPlayer-option.

I opened an issue upstream on jPlayer to discuss it. Depending on their answer, I will fix it in Ampache directly.

- [ ] Add new keyboards bindings
- [ ] Add documentation about keybindings

**Comment by Phyks** [comment]
Also Ctrl+F display visualization full screen when the player is shown, and conflicts with the binding to search within page.

**Comment by roba91** [comment]
Same for `Ctrl+P`.

**Comment by roba91** [comment]
> I opened an issue upstream on jPlayer to discuss it. Depending on their answer, I will fix it in Ampache directly.

JPlayer seems abandoned to me...

**Comment by Phyks** [comment]
Yes, it is. See [this discussion](https://github.com/ampache/ampache/issues/1127#issuecomment-233349698).

**Comment by AshotN** [comment]
I will have this added to the React client, I will avoid custom shortcuts like 

> 
>     * `Ctrl+→` / `Ctrl+←` for next/previous song
> 
>     * `Ctrl+↑` / `Ctrl+↓` for volume up/down
> 
>     * `Space` for play/pause (Or `Ctrl+Space`)
> 

Mainly, these types of shortcuts will only work when the webpage is focused. The React client is already utilizing the [MediaSession](https://developer.mozilla.org/en-US/docs/Web/API/MediaSession) Web API, so those shortcuts are available natively on any browser that supports them. So skip, pause/play, will work like any other native application.

Controls such as Ctrl-F. 

I don't know if anyone has any interest in implementing the MediaSession API or any other shortcuts into the Reborn theme, so I will assign this issue to myself.
